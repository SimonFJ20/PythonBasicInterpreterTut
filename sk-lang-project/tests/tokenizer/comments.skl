capture this
// done capture this
capture this
/*
dont caputure this
*/ capture this
capture this /* but not this */
