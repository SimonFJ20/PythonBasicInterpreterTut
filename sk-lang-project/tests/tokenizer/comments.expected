capture
this
capture
this
capture
this
capture
this
